modules/custom/az_global_footer/data/az_global_footer.json

```
@@ -393,14 +393,14 @@
       "link_id": "36",
       "parent_link_id": "0",
       "menu": "az-footer-social-media",
-      "title": "Twitter",
-      "urlpath": "https://twitter.com/uarizona",
+      "title": "X, formerly known as Twitter",
```

```suggestion
      "title": "X, formerly Twitter",
```
Per @HenryzGreenberg